adoc/Service-Broker-Setup.adoc

```diff
@@ -37,9 +37,20 @@ a {product} cluster.
 - Helm must be configured. For more information, see the
   link:https://docs.helm.sh/using_helm/#quickstart[Helm documentation].
 - A working {product} deployment.
-- External databases must be reachable from the running applications. For
-  more information, see the
+- An Application Security Group (ASG) for applications to reach external databases. For more information, see the 
   link:http://docs.cloudfoundry.org/concepts/asg.html[CloudFoundry Documentation regarding application security groups].
++
+For testing purposes use the following (insecure) workaround:
```

Comment:
This is not a workaround, it's just a permissive security group. Framing the suggestion this way makes it sound like there is a bug.

Reply:
True - I just pushed that one ...